synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -178,9 +178,32 @@ ffmpeg_trgt::init(ProgressCallback* cb = nullptr)
 	// Windows always have ffmpeg
 	ffmpeg_binary_path = synfig::OS::get_binary_path().parent_path() / filesystem::Path("ffmpeg.exe");
 	if (!FileSystemNative::instance()->is_file(ffmpeg_binary_path.u8string())) {
-		synfig::error("Expected FFmpeg binary not found: %s", ffmpeg_binary_path.u8_str());
+        synfig::error("Expected FFmpeg binary not found at expected path: %s, application will try to look at other paths", ffmpeg_binary_path.u8_str());
 		ffmpeg_binary_path.clear();
 	}
+
+    const char* path_env = std::getenv("PATH"); // fetching all the path from the ENVIRONMENT path variable
+    std::vector<std::string> paths;
+    if(ffmpeg_binary_path.empty()){
+        if (path_env) {
+            std::stringstream ss(path_env);
+            std::string dir;
+            char PATH_SEPARATOR = ';';
+            while (std::getline(ss, dir, PATH_SEPARATOR)) { // trying to seach ffmpeg in al
+                synfig::info("searching for ffmpeg exe in -> %s",dir.c_str());
+                ffmpeg_binary_path = filesystem::Path(dir) + filesystem::Path("\\ffmpeg.exe");
```

Comment:
- change the operator: instead of 'concatenator' `+`, use path append `/` (or `.append()`)
- remove the `\\` from the `ffmpeg.exe` path
- Probably `dir` should be used as this: `filesystem::Path::from_native(dir)` .
Please test it placing a valid ffmpeg.exe file in a folder with at least one non-ascii character in its name, and make this folder first in the envvar PATH.

So the line should be:
`ffmpeg_binary_path = filesystem::Path::from_native(dir) / filesystem::Path("ffmpeg.exe");`

Reply:
used the path append operator as well as from_native function
also modified the code to process non ascii values